spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/dao/JdbcTaskExecutionDao.java

```diff
@@ -79,6 +85,12 @@
 	private static final String FIND_TASK_EXECUTIONS = "SELECT TASK_EXECUTION_ID, "
 			+ "START_TIME, END_TIME, TASK_NAME, EXIT_CODE, "
 			+ "EXIT_MESSAGE, LAST_UPDATED, STATUS_CODE "
+			+ "from %PREFIX%EXECUTION order by START_TIME desc, TASK_EXECUTION_ID desc "
+			+ "LIMIT ? OFFSET ?";
```

Comment:
Have you tested this against Oracle?  I didn't think they supported `LIMIT` `OFFEST` syntax.